src/main/java/org/testng/annotations/Ignore.java

```diff
@@ -0,0 +1,20 @@
+package org.testng.annotations;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PACKAGE;
+import static java.lang.annotation.ElementType.TYPE;
+
+/**
+ * Alternative of @Test(enable=false)
+ *
+ * Notice that @Ignore on a class will disable all test methods of the class.
+ *
+ */
+@Retention(java.lang.annotation.RetentionPolicy.RUNTIME)
+@Target({METHOD, TYPE, PACKAGE})
```

Comment:
In fact, 2 years ago, the idea was to have a symmetric annotation to `@Test`.

Yesterday, I added package level because a second person asked for it.

And true, at the method level `@Ignore` is an alias for `@Test(enabled=false)` but at the class level, it changes the enabled status for all `@Test` of the class.